### PR TITLE
DOCS-4490 fix monolog syntax example

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?

Adds monolog v2 example 

### Motivation

DOCS-4490/github issue

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/other_telemetry/connect_logs_and_traces/php.md [hunk 1]
```diff
@@ -67,6 +67,21 @@ If the logger implements the [**monolog/monolog** library][4], use `Logger::push
...
+
+  $logger->pushProcessor(function ($record) {
+      $context = \DDTrace\current_context();
+      return $record->with(message: $record['message'] . sprintf(
+          ' [dd.trace_id=%s dd.span_id=%s]',
+          $context['trace_id'],
+          $context['span_id']
+      ));
+  });
+```
```
Comment (bwoebi) [inline content/en/tracing/other_telemetry/connect_logs_and_traces/php.md:83]: I think we should try to fit it stylistically in line with the other examples, i.e. without extra newline after `<?php` and a closing `?>`.

File: content/en/tracing/other_telemetry/connect_logs_and_traces/php.md [hunk 2]
```diff
@@ -67,6 +67,21 @@ If the logger implements the [**monolog/monolog** library][4], use `Logger::push
 ?>
 ```
 
+For monolog v2:
+
+```php
+<?php
+
```
Comment (kayayarai) [inline content/en/tracing/other_telemetry/connect_logs_and_traces/php.md:74]:
```suggestion
<?php
```

File: content/en/tracing/other_telemetry/connect_logs_and_traces/php.md [hunk 3]
```diff
@@ -67,6 +67,21 @@ If the logger implements the [**monolog/monolog** library][4], use `Logger::push
...
+  $logger->pushProcessor(function ($record) {
+      $context = \DDTrace\current_context();
+      return $record->with(message: $record['message'] . sprintf(
+          ' [dd.trace_id=%s dd.span_id=%s]',
+          $context['trace_id'],
+          $context['span_id']
+      ));
+  });
```
Comment (kayayarai) [inline content/en/tracing/other_telemetry/connect_logs_and_traces/php.md:82]:
```suggestion
    });
  ?>
```

File: content/en/tracing/other_telemetry/connect_logs_and_traces/php.md [hunk 4]
```diff
@@ -67,6 +67,21 @@ If the logger implements the [**monolog/monolog** library][4], use `Logger::push
 ?>
 ```
 
+For monolog v2:
+
+```php
+<?php
+
+  $logger->pushProcessor(function ($record) {
```
Comment (kayayarai) [inline content/en/tracing/other_telemetry/connect_logs_and_traces/php.md:75]:
```suggestion
<?php
  $logger->pushProcessor(function ($record) {
```

File: content/en/tracing/other_telemetry/connect_logs_and_traces/php.md (hunk 1 above)
Comment (kayayarai) [reply content/en/tracing/other_telemetry/connect_logs_and_traces/php.md:83]: updated @bwoebi. Look ok?

**Review by apigirl — APPROVED** [review]

File: content/en/tracing/other_telemetry/connect_logs_and_traces/php.md (hunk 1 above)
Comment (bwoebi) [reply content/en/tracing/other_telemetry/connect_logs_and_traces/php.md:83]: Yep :-)

**Pull request merged**